garak/probes/ansiescape.py

```diff
@@ -111,16 +88,80 @@ def __init__(self, config_root=_config):
         for ask in ASKS:
             for high_level_task in HIGH_LEVEL_TASKS:
                 prompts.append(f"{ask} {high_level_task}")
-        for live_payload in LIVE_PAYLOADS:
+        for live_payload in garak.data.ansi.LIVE_PAYLOADS:
             for repeat_stub in REPEAT_STUBS:
                 prompts.append(f"{repeat_stub} {live_payload}")
-        for escaped_payload in ESCAPED_PAYLOADS:
+        for escaped_payload in garak.data.ansi.ESCAPED_PAYLOADS:
             for unescape_stub in UNESCAPE_STUBS:
                 prompts.append(f"{unescape_stub} {escaped_payload}")
 
         self.prompts = prompts
 
 
+class AnsiRawTokenizerHF(garak.probes.Probe):
+    """Probe for checking a tokenizer for potentially malicious ANSI escape sequences
+
+    Returns any tokens bearing ANSI escape sequences that could be used to seize
+    control of anything rendering output. Always returns one token judged non-malicious.
+    """
+
+    lang = "*"
+    tags = ["owasp:llm05"]
+    supported_generators = {"Model", "Pipeline", "OptimumPipeline", "LLaVA"}
```

Comment:
does `llava` support this?

Reply:
yeah, `AutoTokenizer` supports LLaVA models, and the object returned includes a `vocab` supporting the required access pattern